### DISCUSSION TITLE
Can't import MultithreadedSDMolSupplier

### DISCUSSION THREAD

**Discussion by rajathsalegame** [rdkit/rdkit#3847] [Q&A] [answered]
For some reason, running into the error `AttributeError: module 'rdkit.Chem' has no attribute 'MultithreadedSDMolSupplier' with rdkit version `2020.03.01`. I'm able to import with version `2020.09.4`--is this a version issue or are there some other packages I'm missing? Thanks!

**Comment by greglandrum** [comment, marked as answer]
The MultithreadedSDMolSupplier was added in the 2020.09.1 release.
